packages/flutter_tools/lib/src/commands/flutter_command_runner.dart

```diff
@@ -113,6 +113,16 @@ class FlutterCommandRunner extends CommandRunner {
   }
   List<BuildConfiguration> _buildConfigurations;
 
+  String get enginePath {
```

Comment:
Discussed this and decided on a policy of using the first BuildConfiguration having Android as the target platform.